client/src/App.css

```diff
@@ -1,30 +1,27 @@
+@import url("https://fonts.googleapis.com/css2?family=Public+Sans:wght@500;700;900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@700&display=swap");
```

Comment:
Those imports should be added to `index.css` instead of here. In this way, we can use them in the whole project, not just scoped to `App.css`.

Reply:
done